src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,21 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public void register(@RequestBody UserRequestDto requestDto) {
```

Comment:
better returt UserResponseDTO and add a check whether such a user does not exist

Reply:
I was inspired by common mistakes) I'll fix it now
![image](https://user-images.githubusercontent.com/115244741/235666481-9ebe9044-7553-4ec2-a6d2-410b2e6813c4.png)

Reply:
Also, the email field is unique, is this additional check really necessary?
![image](https://user-images.githubusercontent.com/115244741/235669772-54dfb053-416c-47a5-a71b-f92e23910c97.png)